tests/performance/local/CMakeLists.txt

```
@@ -216,6 +221,11 @@ foreach(benchmark ${benchmarks})
 
 endforeach()
 
+if(HPX_WITH_LIBCDS)
+  target_compile_definitions(cds PUBLIC CDS_THREADING_HPX)
+  target_link_libraries(libcds_hp_overhead_test PRIVATE cds hpx_core)
```

```suggestion
  target_link_libraries(libcds_hp_overhead_test PRIVATE cds)
```

No need for `hpx_core`, it should get linked to HPX by `add_hpx_executable` above.